modules/microshift-disabling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+// * microshift_configuring/microshift-disable-lvms-csi-provider-csi-snapshot.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-lvms-csi-snapshot_{context}"]
+= Disabling deployments that run CSI snapshot implementations
+
+Use the following procedure to disable installation of the CSI implementation pods.
+
+[IMPORTANT]
+====
+This procedure is for users who are defining the configuration file before installing and running {microshift-short}. If {microshift-short} is already started then CSI snapshot implementation will be running. Users must manually remove it by following the uninstallation instructions.
+====
+
+[NOTE]
+====
+{microshift-short} will not delete CSI snapshot implementation pods. You must configure {microshift-short} to disable installation of the CSI snapshot implementation pods during the startup process.
+====
+
+.Procedure
+
+. Disable installation of the CSI snapshot controller by entering the `optionalCsiComponents` value under the `storage` section of the {microshift-short} configuration file in `/etc/microshift/config.yaml`:
++
+[source,yaml]
+----
+# ...
+  storage: {} <1>
+# ...
+----
+<1> Accepted values are:
+* Not defining `optionalCsiComponents`.
+* Specifying `optionalCsiComponents` field with an empty value ([])
```

Comment:
```suggestion
* Specifying `optionalCsiComponents` field with an empty value (`[]`) or a single empty string element (`[""]`)
```
Wrapped the elements with back-ticks to highlight that these are the field values, like the `snapshot-controller` et. al. below. I had to add the alternate value because of a very irritating behavior in the config generator tool we use.

Reply:
done